### ISSUE TITLE
Regarding the instruction of the demo

### ISSUE THREAD

**Issue by Lee-Juntong** [nus-cs2113-AY2021S1/forum#127]
![image](https://user-images.githubusercontent.com/55589340/98685516-6a4c6e00-23a2-11eb-898b-f4d4ba643a66.png)
Should this line be interpreted as we should have 1~2 negative examples per function or in total for the whole demo?

**Comment by okkhoy** [comment]
A few thought the demo is OK. Definitely no more than 1-2 per feature.

**Comment by Lee-Juntong** [comment]
> A few thought the demo is OK. Definitely no more than 1-2 per feature.

Does it mean we can have none for some features?

**Comment by okkhoy** [comment]
Yes. You can do the demo without showing negative examples also.
